fizzbuzz.js

```diff
@@ -0,0 +1,22 @@
+function fizzBuzz(num) {
+
+    for (var i = 0; i <= num; i++) {
+
+        if (num % 3 === 0 && num % 5 === 0) {
+            return "FizzBuzz";
```

Comment:
if you are returning immediately in your for loop, how many times is it executing? You aren't even using `i`